Rocket.Chat/Controllers/Chat/ChatViewController.swift

```diff
@@ -375,6 +383,12 @@ final class ChatViewController: SLKTextViewController {
     }
 
     // MARK: Subscription
+    
+    func updateSubscriptionDraft(message: String) {
+        let userDefaults = UserDefaults.standard
+        userDefaults.set(message, forKey: subscription.rid)
+        userDefaults.synchronize()
```

Comment:
Why do you need to call `synchronize()` here? I believe it should be handled automatically.

Reply:
Actually this call is useless and shouldn't be used, my bad. I'm going to fix it too.